### PR TITLE
Gogiputtar/fix impedance ex

### PR DESCRIPTION
**Pull request by GogiPuttar**
Created four new examples for impedance control, tested on an A-2085-06 arm. 
Version 2.7.7 of hebi-py required since 2.7.6 has faulty impedance control.

### CODE REVIEW

**Comment by GogiPuttar** [comment]
Split ex_impedance_control.py into four separate examples scripts

**Comment by GogiPuttar** [comment]
Tested on 6-DoF T-series arm. All examples work with no issues. Even the impedance controller gains don't need to be changed.

**Comment by GogiPuttar** [comment]
I need to add another example and restructure the floor example so if you could put merging this on hold for a bit, it would be great :)

**Comment by GogiPuttar** [comment]
Example's been added. PR is ready :)

File: kits/arm/ex_impedance_control_damping.py [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+        for i in range(len(lower_limits)):
+
+            if ee_pose_curr[2,3] > lower_limits[i]:
+
+                mode = i
+        
+        # Change gains only upon mode switches
+        if not mode == prevmode and mode >= 0:
+
+            impedance_controller.set_kp(damping_kp[mode][0], damping_kp[mode][1], damping_kp[mode][2], damping_kp[mode][3], damping_kp[mode][4], damping_kp[mode][5])
```
Comment (cwbollinger) [inline kits/arm/ex_impedance_control_damping.py:172]: I would use the spread operator here to make things less verbose:
```
impedance_controller.set_kp(*damping_kp[mode])
impedance_controller.set_kd(*damping_kd[mode])
```

**Review by cwbollinger — CHANGES_REQUESTED** [review]
This looks really good. I commented on a minor tweak to make one section less verbose, but overall this is almost good to go. The only thing I would do is take the matplotlib code, which is shared verbatim, wrap it up in a function that takes a log, and import it from a new file `plotting.py`. Since all these demos are viewing the same values I think it makes sense to avoid duplicating the plotting code.

So in each demo the logic will be
```
if enable_logging:
    hebi_log = arm.group.stop_log()
    draw_plots(hebi_log)
```

File: kits/arm/ex_impedance_control_damping.py (hunk 1 above)
Comment (GogiPuttar) [reply kits/arm/ex_impedance_control_damping.py]: All done!

**Review by cwbollinger — APPROVED** [review]

**Pull request merged**